WordPress/Classes/ViewRelated/Media/MediaLibraryPickerDataSource.m

```diff
@@ -532,6 +532,53 @@ - (WPMediaRequestID)imageWithSize:(CGSize)size completionHandler:(WPMediaImageBl
     return [self.mediaID intValue];
 }
 
+- (WPMediaRequestID)videoAssetWithCompletionHandler:(WPMediaAssetBlock)completionHandler
+{
```

Comment:
I guess the thing is that there is no code on the left? Everything is in an if statement, and they are the three things that can go wrong: it's not a video (can this actually happen? Isn't it a callback?), it has no URL, it has no data. Then all the errors are the same, bar text, which depends on what's wrong? Maybe more comments would help. Or not repeating the error creation code as much.